Sources/MockoloFramework/Operations/UniqueModelGenerator.swift

```diff
@@ -71,7 +71,14 @@ private func generateUniqueModels(key: String,
         mockInheritedTypes.append(.uncheckedSendable)
     }
 
-    let resolvedEntity = ResolvedEntity(key: key, entity: entity, uniqueModels: uniqueModels, attributes: attributes, inheritedTypes: mockInheritedTypes)
+    let resolvedEntity = ResolvedEntity(
+        key: key,
+        entity: entity,
+        uniqueModels: uniqueModels,
+        attributes: attributes,
+        inheritedTypes: mockInheritedTypes,
+        inheritsActorProtocol: inheritedTypes.contains(.actorProtocol)
```

Comment:
This is the most important logic in this PR.
Sorry for the mix of non-essential changes (e.g. renaming from `Type` to `SwiftType`)

Reply:
In this scope, `inheritedTypes` contains all inherited protocols through parent protocol.